src/cf-forms/src/atoms/legend.less

```diff
@@ -0,0 +1,10 @@
+.a-legend {
+    .h4();
+
+    // Legends do not wrap in IE.
+    // Different styles are required to ensure wrapping in different versions.
+    box-sizing: border-box; // IE9-11 & Edge 12-13
+    display: table; // IE8-11
```

Comment:
Yeah, IE10 and 11 are not targetable with conditional comments and from what I've read, it's a messy road trying to make it happen.